packages/ui/src/kit/VInput/VInput.vue

```diff
@@ -46,35 +52,38 @@ export default {
   props: {
     label: {
       type: String,
-      default: null,
+      default: '',
     },
+
+    labelTooltip: {
```

Comment:
Just `tooltip`

Reply:
nope, `tooltip` is too abstract name

Reply:
for example, if you need change icon for tooltip, you need pass `tooltipIcon` prop or make `tooltip` prop as object with passing data like
```
tooltip: {
 label: 'some label',
 icon: 'icon-name',
}
```

Reply:
It scope of input `tooltip` is not so abstract as you think, because input can has only one tooltip, without icons and other things.

Reply:
@lamartire @ifiri for this feature `tooltip` only has text, but in next feature `tooltip` can has icon aw well (for example, question, info, different color and other). For this case, we will be able to make:
```js
tooltip-label
tooltip-icon (current icon as default)
tooltip-color
tooltip-...
```

Reply:
@lamartire @ifiri @kazurus 
also maybe we need define position for tooltip `left` or `right` if input label will be near of the edge of document/window
and prefer use `tooltip-position` for this is better

Reply:
What about use only `tooltip` prop as object with strictly predefined format? I like this way, it's clean, understandable, we can use computed or `data` and 1 line in layout instead of 5-6 lines, it helps keep layout pure. This way uses in Vuetify, for example.

Because input component have too many props even without tooltip.

Reply:
@ifiri in this case we also can use computed and b-bind=tooltipComputed